tools/packaging/static-build/cache_components.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../scripts/lib.sh"
+source "${script_dir}/qemu/build-static-qemu-cc.sh"
+
+export KATA_BUILD_CC="${KATA_BUILD_CC:-}"
+
+if [ "${KATA_BUILD_CC}" == "yes" ]; then
+	export qemu_cc_tarball_name="kata-static-qemu-cc.tar.gz"
+else
+	export qemu_cc_tarball_name="kata-static-qemu-cc.tar.gz"
```

Comment:
It sets the same value to `qemu_cc_tarball_name` whether `KATA_BUILD_CC` or not.

Reply:
Nice catch!